packages/trpc/server/routers/viewer/organizations/create.handler.ts

```diff
@@ -69,121 +69,120 @@ export const createHandler = async ({ input, ctx }: CreateOptions) => {
 
   const t = await getTranslation(ctx.user.locale ?? "en", "common");
   const availability = getAvailabilityFromSchedule(DEFAULT_SCHEDULE);
-  let isOrganizationConfigured = false;
 
-  if (check === false) {
-    isOrganizationConfigured = await createDomain(slug);
+  if (check) {
+    await checkOrg(adminEmail, input.language);
+    return { checked: true };
+  }
+
+  const isOrganizationConfigured = isPlatform ? true : await createDomain(slug);
```

Comment:
this should work too 
` const isOrganizationConfigured = isPlatform || (await createDomain(slug));`

Reply:
Even though proposed code is shorter and checks out its for my brain easier to comperhend one with ternary operators so will leave it.